neutron/templates/configmaps/neutron-openvswitch-agent.sh.yaml.j2

```diff
@@ -6,4 +6,5 @@ data:
     set -x
 
     ovs-vsctl add-br br-ex
+    chown neutron: /run/openvswitch/db.sock
```

Comment:
This is bad workaround as when ovs db server is restarted socket perms are again wrong :(